### PR TITLE
[Obdach] Add new icons for poicategory

### PR DESCRIPTION
**Pull request by JoeyStk**
### Short description
This PR adds six new icons for poicategory

### Proposed changes

- Add svgs for the categories. I adapted the svgs a bit to fit the circle styling of the other svgs, as seen in the PR for the education category. I hope that was correct.
- I pushed a branch "feature/add_new_poi_categories" where I also created the POI categories. As far as my understanding goes, we only need the icons for now (therefore this new branch)

### Side effects

- None I can think of

### Resolved issues

Fixes: #2270

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

**Comment by timoludwig** [comment]
> I pushed a branch "feature/add_new_poi_categories" where I also created the POI categories. As far as my understanding goes, we only need the icons for now (therefore this new branch)

This is a misunderstanding, the constants are not the poi categories, but only the choices for the icons. So yes, they need to be included in this pr, otherwise the icons are not usable.

And no, we don't need the actual poi categories in the repository, those are created in the database.

**Comment by JoeyStk** [comment]
Thank you for clarifying. So it would have been better to use the other branch.

> And no, we don't need the actual poi categories in the repository, those are created in the database.

I'm not completely sure I understand correctly. For the icon names we still need a migration file, right?
If I'm reading this correctly I would add the icon names to the constants, add a migration file, extend the translation file and add a release note. Is that correct?

**Comment by timoludwig** [comment]
> I'm not completely sure I understand correctly. For the icon names we still need a migration file, right? If I'm reading this correctly I would add the icon names to the constants, add a migration file, extend the translation file and add a release note. Is that correct?

Yes, I think that's it.

**Comment by JoeyStk** [comment]
> Yes, I think that's it.

Okay, everything should be uploaded now :)

File: integreat_cms/release_notes/current/unreleased/2270.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+en: Add new POI category icons
+de: Füge neue Icons für POI-Kategoriesn hinzu
```
Comment (timoludwig) [inline integreat_cms/release_notes/current/unreleased/2270.yml:2]:
```suggestion
de: Füge neue Icons für POI-Kategorien hinzu
```

**Review by timoludwig — APPROVED** [review]
Thanks! :+1:

File: integreat_cms/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -1426,6 +1446,10 @@ msgstr "Dienstleistung"
 msgid "Shopping"
 msgstr "Einkaufen"
 
+#: cms/constants/poicategory.py
+msgid "Structure"
+msgstr "Stuktur"
```
Comment (ulliholtgrave) [inline integreat_cms/locale/de/LC_MESSAGES/django.po:1451]:
```suggestion
msgstr "Struktur"
```

**Review by ulliholtgrave — CHANGES_REQUESTED** [review]
Please fix the typo and we are good to go :+1:

File: integreat_cms/locale/de/LC_MESSAGES/django.po (hunk 2 above)
Comment (JoeyStk) [reply integreat_cms/locale/de/LC_MESSAGES/django.po]: Thank you, good catch :facepalm:

**Review by ulliholtgrave — APPROVED** [review]
Thanks, Looks good

**Pull request merged**